packages/core/src/request.ts

```diff
@@ -35,8 +35,7 @@ export function sessionToSentryRequest(session: Session | SessionAggregates, api
     ...(sdkInfo && { sdk: sdkInfo }),
     ...(!!api.tunnel && { dsn: api.dsn.toString() }),
   });
-  // I know this is hacky but we don't want to add `session` to request type since it's never rate limited
-  const type: SentryRequestType = 'aggregates' in session ? ('sessions' as SentryRequestType) : 'session';
+  const type: SentryRequestType = 'session';
```

Comment:
This is ternary is needed for node as `sendSession` is typed as so:

`sendSession?(session: Session | SessionAggregates): PromiseLike<Response>;`

Reply:
😞